velox/expression/ExprCompiler.cpp

```diff
@@ -111,18 +124,27 @@ bool isCall(const TypedExprPtr& expr, const std::string& name) {
   return false;
 }
 
-// Flattens nested ANDs or ORs into a vector of conjuncts
+// Recursively flattens nested ANDs, ORs or eligible callable expressions into a
```

Comment:
It would be nice to update documentation as well: https://facebookincubator.github.io/velox/develop/expression-evaluation.html#flatten-concat-like-functions